content/50.discussion.md

```diff
@@ -11,8 +11,42 @@ In this review, we seek to explain the scientific rationale underlying these tec
+By being self-aware of our own implicit biases, one can work towards equality.
 
-<!--Risk of comorbid health conditions associated with more severe outcomes may be influenced by long-term damage caused by chronic stress related to traumatic social experiences [@doi:10.1007/s12170-013-0338-5], perhaps mediated by cardiovascular risk factors [@doi:10.1101/2020.05.10.20097253], although the effects of chronic stress have not yet been researched in the specific case of COVID-19 disparities.-->
+The population of the inhabitants or humans on Earth is approximately 7.8 billion people.
+The myriad of cultures, languages, and environments differentiate the population into subpopulations with varying beliefs, values, and point of views.
+However, the preponderance of the academic discourse made public is produced by western, educated, industrialized, rich and democratic, or WIERD, individuals, which comprise 12 percent of world occupants.
+While there is not much empirical data on the representativeness of WEIRD individuals within mankind, this blatant disparity brings forth the notion of bias in science, for researchers generalize their discoveries to the entire human population.
+The purpose of acknowledging the implicit biases within academia is to provide an appropriate lens for readers to interpret the information.
+
+Humans, more specifically researchers and scientists, ought to examine their ability to distinguish between evolutionary aspects of psychology and cultural aspects of psychology.
+Evolutionary theory suggests that all human psychology behaves in a similar manner.
+Thus, researchers extrapolate the idea that diversity (race, gender, and etc.) lacks importance in how they conduct their studies.
+For instance, many psychology studies utilize university college students enrolled in a psychology course for their surveys or as subjects.
+These biased results from these studies portray a false representation of global human behavior.
+An individual's environment and culture are paramount to their principles and perception, which gravely affect behavior and, consequently, scientific data.
+
+The role of bias in information and its implications goes beyond psychological studies as it is applicable to medicine: how diseases and illnesses are diagnosed, the varying degrees of treatment, and who are in clinical trials.
+The racism within medicine is critical to understanding how practitioners and researchers’ bias negatively affects health outcomes.
+There is a misconception that African-American individuals have thicker skin and exorbitant pain tolerance in comparison to other demographics.
+As a result, African-Americans can be misdiagnosed and be given incorrect prescriptions.
+The glomerular filtration rate (GFR) is a tool used to measure a specific protein in the blood, which is an indicator of kidney function.
+This sophisicalted equation inputs data, such as, weight, age, sex, serum levels of creatinine, and race, to obtain a result.
```

Comment:
```suggestion
This sophisticated equation inputs data, such as, weight, age, sex, serum levels of creatinine, and race, to obtain a result.
```

Reply:
I think that it is important to have citations for these statements, in the context of a review article.  I also would recommend changing the word "exorbitant" because I don't think it actually means "high" in general;  I think it means "overly expensive" which doesn't make sense here.  Maybe you mean "extreme" or just "unusually high"?
I'm not sure how pain perception in particular fits in with COVID-19 though.  It might make sense to talk about discrimination in the medical system as a whole, but it would be important to structure the text in such a way that it's easy to see the point you're making ("there's been bias on other topics in the past, so maybe there will be bias on this topic now").

Reply:
Yes, I forgot to add citations. 
I was attempting to provide some examples to the extent of discrimination in the medical field, however, I understand they do not relate to covid. Thus, I will try to speak more to the medical field as a whole. One of my concerns is due to the lack of info on covid,, much of what i would be saying would be inferences, correct?